

x_min: -4.0
x_max: 0.0
y_min: -2.5
y_max: 2.5
z_min: -0.6
z_max: 2.5

ransac_threshold: 0.01

sor_mean_k: 50
sor_std_dev: 1.0